shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+
+# TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
+
+import subprocess
+
+from shimmingtoolbox import __dir_config_dcm2bids__
```

Comment:
https://setuptools.readthedocs.io/en/latest/pkg_resources.html

Reply:
@kousu are you suggesting to move the declarations of __xxx__ from the __init__ to the setup.py, so these objects can be called anywhere else without having to be imported? as in [this section](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#environment-objects)